packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasEstimates.ts

```diff
@@ -11,17 +12,19 @@ async function fetcher([
   destinationChainId,
   sourceChainErc20Address,
   destinationChainErc20Address,
-  amount
+  amount,
+  balance
 ]: [
   signer: Signer,
   sourceChainId: number,
   destinationChainId: number,
   sourceChainErc20Address: string | undefined,
   destinationChainErc20Address: string | undefined,
-  amount: BigNumber
+  amount: BigNumber,
+  balance: BigNumber
 ]): Promise<GasEstimates | DepositGasEstimates | undefined> {
   // use chainIds to initialize the bridgeTransferStarter to save RPC calls
-  const bridgeTransferStarter = await BridgeTransferStarterFactory.create({
+  const bridgeTransferStarter = BridgeTransferStarterFactory.create({
```

Comment:
Why are we missing `await` here?

Reply:
it's not needed. `create` is not an async function
![image](https://github.com/OffchainLabs/arbitrum-token-bridge/assets/5085333/901f14f0-129b-4923-a46a-e282e87ca6aa)

Reply:
Not a big deal, but something we might want to add to our eslint config